add documentation for solr-build-index (#38)
opendata-swiss/ogd-handbook-wiki: library/advanced-search.en.md

By default, all the fields that belong to a dataset are copied in one field (called _"text"_), so that the search process only has to check one field to find a match.
So if a user submits a search with the query "weather", Solr runs this query against the _"text"_ field of all datasets.

<a name="search-index"></a>
## Search Index
To build the search index, various fields will be queried. In general, following fields will be indexed:

URLs:
* `url`
* `ckan_url`
* `download_url`
* `res_url`

Text-fields:
* `extras_*`
* `res_extras_*`
* `urls`
* `name`
* `title`
* `title_string`
* `text`
* `license`
* `notes`
* `tags`
* `groups`
* `organization`
* `res_name`
* `res_format`
* `res_description`
* `identifier`
* `see_alsos`
* `maintainer`
* `author`
* `publishers`
* `contact_points`

Translated fields:
* `title`
* `keywords`
* `groups`
* `organization`
* `res_name`
* `res_description`

> More detailed information can be found in the config-fields for Solr which can be found here and in the [official Solr documentation](https://lucene.apache.org/solr/guide/6_6/index.html):

Reviewer: ```suggestion
Find more detailed information about the Solr configuration in the [official Solr documentation](https://lucene.apache.org/solr/guide/6_6/index.html). The config and schema of opendata.swiss is available on GitHub:
```

* [solrconfig.xml](https://github.com/opendata-swiss/ckanext-switzerland/blob/master/solr/solrconfig.xml)
* [schema.xml](https://github.com/opendata-swiss/ckanext-switzerland/blob/master/solr/schema.xml)

> The source of the referenced files in the `solr.xml` (e.g. `italian_stop.txt`, `fr_elision.txt`, etc.) can be found in the official CKAN-Repository of the current CKAN-Version on Github [Solr-Folder in CKAN-Repository](https://github.com/ckan/ckan/tree/master/ckanext/multilingual/solr). All files that can not be found in this folder (e.g. `stopwords.txt`) are the default-files provided by Solr.

Reviewer: ```suggestion
The source of the referenced files in the `solr.xml` (e.g. `italian_stop.txt`, `fr_elision.txt`, etc.) can be found in the official CKAN-Repository of the current [CKAN-Version on Github](https://github.com/ckan/ckan/tree/master/ckanext/multilingual/solr). All other files (e.g. `stopwords.txt`) are provided by Solr.
```
